README.md

```diff
@@ -9,15 +9,42 @@ git submodule update
 # Install tcmalloc according to
 # https://github.com/gperftools/gperftools/blob/master/INSTALL
 
-pushd redis && env USE_TCMALLOC=yes make -j && popd
-pushd glog && cmake . && make -j install && popd
-pushd leveldb && make -j && popd
+cd redis
+env USE_TCMALLOC=yes make -j
+cd ..
+
+cd glog 
+cmake . 
+make -j install 
+cd ..
+
+cd leveldb 
+make -j 
+cd ..
+
+# Install grpc. A suitable version is included as a submodule of etcd3 but you
+# can install the latest version separately on the system too.
+cd etcd3
+git submodule update --init
+cd grpc 
+git submodule update --init 
+make 
+make install
+cd ../..
+```
+
+## etcd
+To make etcd-related tests pass and experiment with the etcd master, you
+need to have etcd running locally, listening on 127.0.0.1:12379.
 
-mkdir build; cd build
-cmake ..
-make -j
+A really simple way to do this is to install Docker, and then:
+
+```
+docker run -d -p 12379:2379 appcelerator/etcd
```

Comment:
If not too hard, can you also describe a way to manually start etcd without relying on docker?